plat/common/include/uk/plat/common/irq.h

```diff
@@ -59,4 +59,13 @@ enum uk_irq_polarity {
 	UK_IRQ_POLARITY_MAX
 };
 
+/**
+ * Calls the registered IRQ handlers for the given IRQ vector. This will
+ * acknowledge the IRQ.
+ *
+ * @param irq IRQ vector
+ */
+void _ukplat_irq_handle(unsigned long irq);
+void _ukplat_irq_handle(unsigned long irq);
```

Comment:
Hey @kubanrob,

thanks for updating! Good solution to the GIC switch. I noticed that this line is duplicated, though.